jerry-core/ecma/base/ecma-helpers-string.cpp

```diff
@@ -993,6 +993,36 @@ ecma_string_to_number (const ecma_string_t *str_p) /**< ecma-string */
 } /* ecma_string_to_number */
 
 /**
+ * Check if string is array index.
+ */
+bool
+ecma_string_get_array_index (const ecma_string_t *str_p, /**< ecma-string */
+                             uint32_t *index) /**< index */
```

Comment:
Please, add `out: ` mark to the comment, and rename the parameter to `out_index_p`.